backend/src/plugins/Tags/types.ts

```diff
@@ -16,6 +16,7 @@ export const TagCategory = t.type({
   user_category_cooldown: tNullable(t.union([t.string, t.number])), // Per user, per tag category
   global_tag_cooldown: tNullable(t.union([t.string, t.number])), // Any user, per tag
   global_category_cooldown: tNullable(t.union([t.string, t.number])), // Any user, per category
+  category_delete_invoke: tNullable(t.boolean), // Any tag, per tag category
```

Comment:
I don't think this needs the `category` prefix, it's only needed for cooldowns as those have a difference betwee per tag and per category.